lib/js/importers/AllInOneImporter.ts

```diff
@@ -0,0 +1,189 @@
+const fsT = require('fs');
+const axios = require("axios");
+const tokensFilesConfig = JSON.parse(
+	fsT.readFileSync('lib/js/importers/configs/allInOneConfig.json', 'utf8')
+);
+
+interface IResultJsonObject {
+	id: string;
+	label: string;
+	value: string;
+	weight: number;
+}
+
+interface configFileObject {
+	destination: string,
+	destinationJson: string
+}
+
+interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+const ImportColorsRaw = (
+	binValues: configFileObject,
+	jsonSource: { data }
+) => {
+	let hexToCssVariable: Array<Object> = [];
+	try {
+		let result: Array<string> = [];
+		let resultJson = {};
+
+		jsonSource.data.colors.forEach(obj => {
+			const patternIsRawColor = /RAW\/|theme/i;
```

Comment:
variable name doesn't match what we check inside. The same for function name

Reply:
Updated.

Reply:
Looks better. Now it's consistent with what's inside. IMO, it could be even better, if we name it based on it's function, like `patternColorsToProcess` and `patternColorsToIgnore`. If we do that, you already know how to use this variable and adding another pattern won't require variable rename.

Reply:
Sure, sounds clear.